run.go

```diff
@@ -43,6 +44,13 @@ func CurrentFPS() float64 {
 	return clock.CurrentFPS()
 }
 
+func SetBenchmark(b *testing.B) {
+	g, ok := theGraphicsContext.Load().(*graphicsContext)
```

Comment:
It looks like calling `SetBenchmark` before `Run` is ignored. This is a little counterintuitive for users. I was wondering if it'd be possible to store the given benchmark somewhere if `theGraphicsContext` is not present and use it when creating `theGraphicsContext`. If the benchmark is stored at a global variable, `graphicsContext` would not need to have a benchmark member.

Reply:
That is a good idea.